File: helix/src/main/java/glitch/helix/object/json/Bits.java
package glitch.helix.object.json;

import com.google.gson.annotations.SerializedName;
import lombok.Data;

@Data
public class Bits {
    private final Long userId;
    @SerializedName("user_name")
    private final String username;
    private final int rank;
    private final int score;
}

File: helix/src/main/kotlin/glitch/helix/object/json/bits.kt
package glitch.helix.`object`.json

import com.google.gson.annotations.SerializedName

/**
 *
 * @author Damian Staszewski [damian@stachuofficial.tv]
 * @version %I%, %G%
 * @since 1.0
 */
data class Bits(
        val userId: Long,
        @SerializedName("user_name")
        val username: String,
        val rank: Int,
        val score: Int
)